internal/jsre/deps/web3.js

```diff
@@ -5372,6 +5372,13 @@ var methods = function () {
         inputFormatter: [formatters.inputTransactionFormatter]
     });
 
+    var fillTransaction = new Method({
+        name: 'fillTransaction',
+        call: 'eth_fillTransaction',
+        params: 1,
+        inputFormatter: [formatters.inputTransactionFormatter]
+    });
```

Comment:
Please move this to `internal/web3ext`

Reply:
All our extensions to web3 are in that file, which ensures we can change web3.js without touching anything else. Not that we will ever do really.